lancet/commands/configuration.py

```diff
@@ -10,60 +10,101 @@
 
 
 @click.command()
-@click.option("-f", "--force/--no-force", default=False)
+@click.option(
+    "-f",
+    "--force/--no-force",
+    default=False,
+    help="Setup even if .lancet already exists.",
+)
+@click.option(
+    "-d",
+    "--debug/--no-debug",
+    default=False,
+    help="Adds additional steps to the wizard for debugging.",
+)
 @click.pass_context
-def setup(ctx, force):
-    """Wizard to create the user-level configuration file."""
-    if os.path.exists(USER_CONFIG) and not force:
+def setup(ctx, force, debug):
+    # environment variable used for testing
+    user_path = os.environ.get("ENV_USER_CONFIG") or USER_CONFIG
```

Comment:
Why the ENV prefix? And this definitely needs a `LANCET` prefix instead.
Also, I'd move it to the settings.py file and make sure that `USER_CONFIG` always reflects the correct value.

Reply:
Switched to specific lancet_test prefix. I tried moving it into the settings.py as it seemed better to me. The problem is that the env variable is loaded inside the setup function and everything outside (settings) are not updated. I wasn't able to get over that obstacle